src/lib.rs

```diff
@@ -64,6 +64,13 @@
 //! let y_hat = knn.predict(&x).unwrap();
 //! ```
 
+#![allow(
```

Comment:
Can you either move this block to the top, right under 
```
#![warn(missing_docs)]
#![warn(missing_doc_code_examples)]
```
or move this block close to yours?